tutorials/mongoose/mongoose-validate-unique-email.md

```diff
@@ -0,0 +1,5 @@
+With Mongoose, you can prevent duplicates in your databases using [Validation](https://mongoosejs.com/docs/validation.html). Validation is defined in the SchemaType and is a middleware. You can also create your own validation in the schema or you can use Mongooses's built in validation. To prevent duplicates, we recommend using the `unique` property as it tells Mongoose each document should have a unique value for a given path, and you can read about that [here](https://masteringjs.io/tutorials/mongoose/unique). It is important to note that the `unique` option is not a validator, and you can read more about that [here](https://mongoosejs.com/docs/validation.html#the-unique-option-is-not-a-validator). It is a shorthand for creating a MongoDB unique index on, in this case, `email`. If you wait for the index to be built, you can use either Mongoose's `Model.on('index')` event or you can you Mongoose's promised based alternative, `Model.init()`, which is shown below::
```

Comment:
1. Lowercase "v" in validation in the first sentence
2. "Read about that here" is not a good pattern for readability or SEO. I'd just get rid of the "you can read about that here" part, and make "`unique` property" link to the unique tutorial
3. Let's move the part about the `unique` option not being a validator below the code sample